.changeset/spicy-wombats-work.md

```diff
@@ -0,0 +1,6 @@
+---
+'@crowdstrike/glide-core': patch
+---
+
+- Link's `href` attribute is now optional to match native.
```

Comment:
Unrelated: Should Menu Link and Split Button Primary Link's `url` attributes be renamed to `href`?